LearningMissionSimulation/PlaygroundMher.cs

```diff
@@ -115,13 +115,25 @@ public void AssignModulesToInstructors()
             foreach (var instructor in instructorList)
             {
                 instructor.ModuleList = GetModuleList();
+                AddToModuleInstructorList(instructor.ModuleList, instructor);                
                 Console.WriteLine("\n========== Generated List Number {0} of the Instructor Number {1} =======\n\n\n", i+1, i+1);
                 i++;
             }
             Console.WriteLine("========== Generated Lists of the Instructors Module List  =======\n");
             Console.WriteLine("******** Finished assigning modules to instructors  ********\n");
         }
 
+        void AddToModuleInstructorList(List<Module> moduleList, Instructor instructor)
```

Comment:
You just need one input parameter: instructor. You will get the module list inside the method.